CHANGELOG.md

```diff
@@ -15,6 +21,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
    + Enables uploading raw text to a function with `Content-Type: text/plain`
 - #1938, Allow escaping inside double quotes with a backslash, e.g. `?col=in.("Double\"Quote")`, `?col=in.("Back\\slash")` - @steve-chavez
 - #1075, Allow filtering top-level resource based on embedded resources filters. This is enabled by adding `!inner` to the embedded resource, e.g. `/projects?select=*,clients!inner(*)&clients.id=eq.12`- @steve-chavez, @Iced-Sun
```

Comment:
Since we're introducing this feature and already know about #2009, we should solve this by at least disabling `count=exact` in the problematic case for this release - or by fixing it alltogether. But it seems odd to release a known-to-be-broken feature.

Reply:
Fair enough! I've opened https://github.com/PostgREST/postgrest/pull/2048 for that.